shared/chat/conversation/header/container.js

```diff
@@ -13,24 +13,24 @@ import type {TypedState} from '../../../constants/reducer'
 type OwnProps = {
   onBack: () => void,
   onToggleSidePanel: () => void,
-  selectedConversation: ?Constants.ConversationIDKey,
+  selectedConversationIDKey: ?Constants.ConversationIDKey,
   sidePanelOpen: boolean,
 }
 
 const getYou = (state: TypedState) => state.config.username || ''
 const getFollowingMap = (state: TypedState) => state.config.following
 const getMetaDataMap = (state: TypedState) => state.chat.get('metaData')
-const getSelectedConversation = (_, selectedConversation: ?Constants.ConversationIDKey) => selectedConversation
-const getSelectedInbox = (state: TypedState, selectedConversation: ?Constants.ConversationIDKey) => (
-  state.chat.get('inbox').find(inbox => inbox.get('conversationIDKey') === selectedConversation)
+const getSelectedConversation = (_, selected: ?Constants.ConversationIDKey) => selected
```

Comment:
I was going through and renaming this from selectedConversation to selectedConversationIDKey and just wanted to reduce how much horizontal space this took in these tiny closures